src/MissionCreator.sol

```diff
@@ -150,6 +156,17 @@ contract MissionCreator is Ownable, IERC721Receiver {
             lockedUntil: 0, // The split is not locked, meaning the project owner can remove it or change it at any time.
             hook: IJBSplitHook(address(0)) // Not used.
         });
+        // amm token split
+        rulesetConfigurations[0].splitGroups[1].splits[2] = JBSplit({
+            percent: 100_000_000, // 10%, out of 1_000_000_000
+            projectId: 0, // Not used.
+            preferAddToBalance: false, // Not used, since projectId is 0.
+            beneficiary: moonDAOTreasuryPayable, // The beneficiary of the split.
+            lockedUntil: 0, // The split is not locked, meaning the project owner can remove it or change it at any time.
```

Comment:
Should maybe be locked? Otherwise the project owner could modify these

Reply:
done